docs/modules/ROOT/pages/reference/cli.adoc

```diff
@@ -0,0 +1,98 @@
+= Command Line Options
+
+== Generic Options
+
+Some command line options aren't associated with a specific command, but can
+provided to any command instead.
+
+*--version*::
+  Show the version and exit.
+
+*-v, --verbose*::
+  Control verbosity. Can be repeated for more verbose output.
++
+Currently three verbosity levels are implemented, regular operation is
+active when the flag isn't provided, debug verbosity is enabled when the
+flag is provided once or more, and trace verbosity is enabled when the flag
+is provided three times or more.
+
+*--help*::
+  Show a help message and exit.
+
+== Catalog Clean
+
+This command doesn't have any command line options.
+
+== Catalog Compile
+
+*--api-url* URL::
+  xref:lieutenant:ROOT:index.adoc[Lieutenant] API URL.
+
+*--api-token* TOKEN::
+  Lieutenant API token.
+
+*--global-git-base* URL::
+  Base directory for global Git config repositories.
```

Comment:
IMHO this needs a bit more explanation what this actually means and what is expected to be found under this Git base, maybe even include an example?